packages/events/src/service/config/config.ts

```diff
@@ -34,6 +38,41 @@ export async function getEventConfigurations(token: string) {
   return array(EventConfig).parse(await res.json())
 }
 
+async function getEventConfigurationById({
+  token,
+  eventType
+}: {
+  token: string
+  eventType: string
+}) {
+  const configurations = await getEventConfigurations(token)
+
+  return configurations.find((config) => config.id === eventType)
+}
+
+export async function getActionFormFields({
+  token,
+  eventType,
+  action
+}: {
+  token: string
+  eventType: string
+  action: ActionType
+}): Promise<FieldConfig[]> {
+  const configuration = getOrThrow(
+    await getEventConfigurationById({
+      token,
+      eventType
+    }),
```

Comment:
Previously the result expectation convention in the codebase has been so that if a method starts with `get..` it's implied to throw an error if said thing is not found. `find..` starting ones are ones that return a null or undefined but do not throw. Let's change this to either internally throw or be a find... function